backend/src/api/controllers/user-controllers.ts

```diff
@@ -0,0 +1,46 @@
+import type { Request } from 'express';
+import { HttpCode, HttpError } from 'growup-shared';
+
+import type {
+  UserLoginForm,
+  UserRegisterForm,
+} from '../../common/forms/user.forms';
+import {
+  authenticateUser,
+  registerUser,
+  getUserJWT,
+} from '../../services/user.service';
+
+type TokenResponse = {
+  token: string;
+};
+
+const loginUserController = async (req: Request): Promise<TokenResponse> => {
+  const userCredentials: UserLoginForm = req.body;
+  const user = await authenticateUser(userCredentials);
+
+  if (!user)
+    throw new HttpError({
+      status: HttpCode.NOT_FOUND,
+      message: 'Wrong credentials',
+    });
+
+  const token = await getUserJWT(user);
```

Comment:
This is the part of business logic and it should take place in user.serivce 
We do not need separate controller files because `src/api/routes`  are controllers themselves. 
Pass `req.body` to user.service from `user-route.ts`